react/features/watchrtc/watchRTCHandler.ts

```diff
@@ -0,0 +1,30 @@
+
+/* eslint-disable lines-around-comment */
+// @ts-expect-error
+import watchRTC from '@testrtc/watchrtc-sdk';
+/* eslint-enable lines-around-comment */
+
+import { IWatchRTCConfiguration } from '../base/config/configType';
+import logger from './logger';
+
+/**
+ * Class that controls the watchRTC flow, because it overwrites and proxies global function it should only be
+ * initialized once.
+ */
+class watchRTCHandler {
```

Comment:
nit. use PascalCase for class name. Also I think there are a couple of linting errors around, consider running the linter.